docs/components/board/jetson.md

```diff
@@ -13,6 +13,9 @@ tags: ["board", "components"]
 
 Follow one of our Jetson [setup guides](/installation/) to prepare your board for running `viam-server` before configuring a `jetson` board.
 
+If after configuring a `jetson` board you want to add your Orin Nano's CSI camera to your robot, you can with Viam's `viam:camera:csi` model.
+Follow [these instructions](/extend/modular-resources/examples/csi/) to do so.
```

Comment:
Does it come with one by default? Otherwise this may be more appropriate:

```suggestion
If you have a CSI camera, follow [these instructions](/extend/modular-resources/examples/csi/) to configure it using the `viam:camera:csi` model.
```

Reply:
Paging @seanavery for confirmation but I assumed it did because the camera was embedded? I could be confused

Reply:
CSI cameras must be purchased separately from the devboard. If it is helpful we can provide a list of commonly used CSI cameras for Jetson Orin-nano platform.